server/monitor.go

```diff
@@ -2357,15 +2357,15 @@ func (s *Server) Jsz(opts *JSzOptions) (*JSInfo, error) {
 		js, cc := s.getJetStreamCluster()
 		if js == nil {
 			// Ignore
-			return nil, ErrJetStreamNotEnabled
+			return nil, fmt.Errorf("%w: no cluster", errSkipZreq)
 		}
 		// So if we have JS but no clustering, we are the leader so allow.
 		if cc != nil {
 			js.mu.RLock()
 			isLeader := cc.isLeader()
 			js.mu.RUnlock()
 			if !isLeader {
-				return nil, errNotLeader
+				return nil, fmt.Errorf("%w: not leader", errSkipZreq)
```

Comment:
Not sure if this will work with the errors.Is above..

Reply:
It works, thats the point of `%w` it wraps the error and errors.Is() checks the wrapped errors.

Reply:
Gotcha